apps/expo/src/components/notif/dmNotifs/dmNotifs.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { render, screen } from '@testing-library/react-native'
+
+import * as api from '~/utils/api'
```

Comment:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/22ef8f36-2c83-4a68-9c2e-4d8e872b20ce)

path for this has been updated

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/f414a6cb-efad-4fc9-9137-b8c04d3cc0a5)